src/Common/Core/Test/Microsoft.Common.Core.Test.csproj

```diff
@@ -44,6 +44,10 @@
       <Project>{01E3D8C2-9C24-492B-B13B-C6D7068B6BF8}</Project>
       <Name>Microsoft.Rtvs.Core.Windows</Name>
     </ProjectReference>
+    <ProjectReference Include="..\..\..\Windows\R\Interpreters\Impl\Microsoft.R.Interpreters.Windows.csproj">
+      <Project>{DE1EC50A-C588-45A9-B22C-981419CD7CD4}</Project>
+      <Name>Microsoft.R.Interpreters.Windows</Name>
```

Comment:
We will have many more of those services when we get to the `RComponents`. We probably need to specify some common entry points in each assembly that provides services, so it can be accessed by both test fixtures and RPackage.